lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -18,9 +21,6 @@ import { ISerializableTokenCache } from "../cache/interface/ISerializableTokenCa
 import { ClientCredentials } from "../account/ClientCredentials";
 import { ProtocolMode } from "../authority/ProtocolMode";
 
-// Token renewal offset default in seconds
-const DEFAULT_TOKEN_RENEWAL_OFFSET_SEC = 300;
```

Comment:
Is this part of the public API? Isn't this a breaking change?

Reply:
It is part of the public api, and it's not a breaking change. I updated the imports in msal-common.

It's used in all clients (public api, msal-common), but Managed Identity (msal-node) needs this constant.